crates/asm-spec/asm.yml

```
@@ -289,6 +303,7 @@ StateRead:
             
             MutKeys:
               opcode: 0x34
+              short: MKEY
```

```suggestion
              short: MKEYS
```

just thinking in case we end up with some op that pushes only one key to the stack somehow?